docs/features/accessibility.md

```diff
@@ -0,0 +1,333 @@
+---
+category: features
+classes: keyboard-shortcuts
+meta-title: Accessibility support | CKEditor 5 Documentation
+modified_at: 2024-03-06
+---
+
+# Accessibility support
+
+CKEditor&nbsp;5 incorporates various accessibility features, including keyboard navigation, screen reader support (<acronym title="Accessible Rich Internet Applications">ARIA</acronym> attributes), and robust semantic output markup. For a more detailed overview and the current status of editor accessibility, please continue reading below.
+
+## Conformance with WCAG 2.x and Section 508
+
+CKEditor 5 is compliant with [Web Content Accessibility Guidelines 2.2](https://www.w3.org/TR/WCAG22/) (<acronym title="Web Content Accessibility Guidelines">WCAG</acronym>) 2.2 levels A and AA and [Section 508 of the Rehabilitation Act](https://www.access-board.gov/ict/) unless stated otherwise in the [Accessibility Conformance Report](#accessibility-conformance-report-vpat).
```

Comment:
```suggestion
CKEditor&nbsp;5 is compliant with [Web Content Accessibility Guidelines 2.2](https://www.w3.org/TR/WCAG22/) (<acronym title="Web Content Accessibility Guidelines">WCAG</acronym>) 2.2 levels A and AA and [Section 508 of the Rehabilitation Act](https://www.access-board.gov/ict/) unless stated otherwise in the [Accessibility Conformance Report](#accessibility-conformance-report-vpat).
```